src/util/ConfigManager/ConfigManager.cpp

```diff
@@ -119,15 +119,16 @@ void ConfigManager::verifyPathToConfigOption(
 }
 
 // ____________________________________________________________________________
-void ConfigManager::addConfigOption(
+const ConfigOption& ConfigManager::addConfigOption(
     const std::vector<std::string>& pathToOption, ConfigOption&& option) {
   // Is the path valid?
   verifyPathToConfigOption(pathToOption, option.getIdentifier());
 
-  // Add the configuration option.
-  configurationOptions_.insert(
-      {createJsonPointerString(pathToOption),
-       std::make_unique<ConfigOption>(std::move(option))});
+  // Add the configuration option and return the inserted elements.
```

Comment:
Just one question:
( Maybe I have only forgotten this)
What happens if you add two options with the same path?
Is this handled by `verifyPathToConfigOption`?

Reply:
`verifyPathToConfigOption` checks, if the given path is already in use and throws a custom exception, if it is, yes.